coalib/results/HiddenResult.py

```diff
@@ -15,15 +18,69 @@ class HiddenResult(Result):
     to transfer any data from a dependent bear to others.
     """
 
-    def __init__(self, origin, contents):
+    @enforce_signature
+    def __init__(self,
+                 origin,
+                 contents,
+                 message: str='',
+                 affected_code: (tuple, list)=(),
+                 severity: int=RESULT_SEVERITY.NORMAL,
+                 additional_info: str='',
+                 debug_msg='',
+                 diffs: (dict, None)=None,
+                 confidence: int=100,
+                 aspect: (aspectbase, None)=None,
+                 message_arguments: dict={},
+                 applied_actions: dict={}):
         """
         Creates a new HiddenResult. The contents can be accessed with
         obj.contents later.
 
-        :param origin:   The originating bear.
-        :param contents: Any object that is picklable since it will be
-                         transferred across processes.
+        :param origin:
+            The originating bear.
+        :param contents:
+            Any object that is picklable since it will be
+            transferred across processes.
+        :param affected_code:
+            A tuple of SourceRange objects pointing to related positions in the
```

Comment:
put `SourceRange` into double backticks ;)
(I know it's copied from `Result`, however let's do it right now :D Filed an issue for this btw: https://github.com/coala/coala/issues/4671)